### PR TITLE
Next 13: Remove anchor tags from Link component

### PR DESCRIPTION
**Pull request by tordans**
Closes: https://github.com/blitz-js/blitz/issues/4066

### What are the changes and their implications?

Change all `<Link><a className="foo">Foo</a></Link>` to `<Link className="foo">Foo</Link>` in this repo.

## Bug Checklist

- [x] Changeset added (run `pnpm changeset` in the root directory)
- [ ] Integration test added (see [test docs](https://blitzjs.com/docs/contributing#running-tests) if needed)

### CODE REVIEW

**Review by siddhsuresh — CHANGES_REQUESTED** [review]
I feel these `passHref` prop are no longer required.

File: .changeset/swift-glasses-laugh.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+"toolkit-app": patch
+"toolkit-app-passport": patch
+"@blitzjs/generator": patch
+"@blitzjs/recipe-ghost": patch
```
Comment (siddhsuresh) [inline .changeset/swift-glasses-laugh.md:5]:
```suggestion
"@blitzjs/generator": patch
```

File: apps/toolkit-app-passportjs/src/auth/components/LoginForm.tsx [hunk 2]
```diff
@@ -41,15 +41,15 @@ export const LoginForm = (props: LoginFormProps) => {
         <LabeledTextField name="password" label="Password" placeholder="Password" type="password" />
         <div>
           <Link href={Routes.ForgotPasswordPage()} passHref>
```
Comment (siddhsuresh) [inline apps/toolkit-app-passportjs/src/auth/components/LoginForm.tsx:43]:
```suggestion
          <Link href={Routes.ForgotPasswordPage()}>
```

File: apps/toolkit-app-passportjs/src/auth/components/LoginForm.tsx [hunk 3]
```diff
@@ -41,15 +41,15 @@ export const LoginForm = (props: LoginFormProps) => {
...
-            <a>Forgot your password?</a>
+            Forgot your password?
           </Link>
         </div>
       </Form>
 
       <div style={{ marginTop: "1rem" }}>
         Or{" "}
         <Link href={Routes.SignupPage()} passHref>
```
Comment (siddhsuresh) [inline apps/toolkit-app-passportjs/src/auth/components/LoginForm.tsx:51]:
```suggestion
        <Link href={Routes.SignupPage()}>
```

File: apps/toolkit-app-passportjs/src/pages/index.tsx [hunk 4]
```diff
@@ -38,15 +38,11 @@ const UserInfo = () => {
   } else {
     return (
       <>
-        <Link href={Routes.SignupPage()} passHref>
-          <a className="button small">
-            <strong>Sign Up</strong>
-          </a>
+        <Link href={Routes.SignupPage()} passHref className="button small">
```
Comment (siddhsuresh) [inline apps/toolkit-app-passportjs/src/pages/index.tsx:41]:
```suggestion
        <Link href={Routes.SignupPage()}className="button small">
```

File: apps/toolkit-app-passportjs/src/pages/index.tsx [hunk 5]
```diff
@@ -38,15 +38,11 @@ const UserInfo = () => {
...
+          <strong>Sign Up</strong>
         </Link>
-        <Link href={Routes.LoginPage()} passHref>
-          <a className="button small">
-            <strong>Login</strong>
-          </a>
+        <Link href={Routes.LoginPage()} passHref className="button small">
```
Comment (siddhsuresh) [inline apps/toolkit-app-passportjs/src/pages/index.tsx:44]:
```suggestion
        <Link href={Routes.LoginPage()} className="button small">
```

File: apps/toolkit-app/src/auth/components/LoginForm.tsx [hunk 6]
```diff
@@ -41,15 +41,15 @@ export const LoginForm = (props: LoginFormProps) => {
         <LabeledTextField name="password" label="Password" placeholder="Password" type="password" />
         <div>
           <Link href={Routes.ForgotPasswordPage()} passHref>
```
Comment (siddhsuresh) [inline apps/toolkit-app/src/auth/components/LoginForm.tsx:43]:
```suggestion
          <Link href={Routes.ForgotPasswordPage()}>
```

File: apps/toolkit-app/src/auth/components/LoginForm.tsx [hunk 7]
```diff
@@ -41,15 +41,15 @@ export const LoginForm = (props: LoginFormProps) => {
...
-            <a>Forgot your password?</a>
+            Forgot your password?
           </Link>
         </div>
       </Form>
 
       <div style={{ marginTop: "1rem" }}>
         Or{" "}
         <Link href={Routes.SignupPage()} passHref>
```
Comment (siddhsuresh) [inline apps/toolkit-app/src/auth/components/LoginForm.tsx:51]:
```suggestion
        <Link href={Routes.SignupPage()}>
```

File: .changeset/swift-glasses-laugh.md (hunk 1 above)
Comment (siddhsuresh) [reply .changeset/swift-glasses-laugh.md:5]: The other 3 packages are ignored by changesets, causing the test failure.

**Comment by tordans** [comment]
@siddhsuresh I apply your review, squashed and force pushed for a clean history. Let me know if I can help with anything else.

**Review by siddhsuresh — APPROVED** [review]

**Comment by Yansongsongsong** [comment]
Hi there. It seems that for initialized scaffold, not only anchor tag in Link component but also issues on SSR and some class name in html labels.

can see the details here: https://github.com/blitz-js/blitz/issues/4066#issuecomment-1409666064

**Comment by tordans** [comment]
@Yansongsongsong but those are separate issues. 
My personal take it: This would go faster, if we split them up in separate issues and if someone would create a PR for fixing them :-).

**Review by flybayer — APPROVED** [review]

**Pull request merged**